packages/application-extension/schema/main.json

```diff
@@ -49,7 +49,103 @@
       "selector": "body"
     }
   ],
-  "properties": {},
+  "jupyter.lab.transform": true,
+  "properties": {
+    "contextMenu": {
+      "title": "The application context menu.",
+      "description": "Note: To disable a context menu item,\ncopy it to User Preferences and add the\n\"disabled\" key. The following example will disable Download item on files:\n{\n  \"contextMenu\": [\n    {\n        \"command\": \"filebrowser:download\",\n        \"selector\": \".jp-DirListing-item[data-isdir=\\\"false\\\"]\",        \"disabled\": true\n        }\n      ]\n\n}\n\nContext menu description:",
```

Comment:
There is a newline missing and some whitespaces can be removed to improve the alignment of closing brackets
```suggestion
      "description": "Note: To disable a context menu item,\ncopy it to User Preferences and add the\n\"disabled\" key. The following example will disable Download item on files:\n{\n  \"contextMenu\": [\n    {\n        \"command\": \"filebrowser:download\",\n        \"selector\": \".jp-DirListing-item[data-isdir=\\\"false\\\"]\",\n        \"disabled\": true\n    }\n]\n\n}\n\nContext menu description:",
```


![Screenshot from 2021-06-20 00-52-42](https://user-images.githubusercontent.com/5832902/122658394-89935100-d164-11eb-9d55-b04998e41e68.png)